### PR TITLE
Updated StatusBar to v4.2.1 (merged master into outsystems branch)

### PR DESCRIPTION
**Pull request by pmaroco**

### CODE REVIEW

**Review by bgracio — APPROVED** [review]

File: src/android/StatusBar.java [hunk 1]
```diff
@@ -214,6 +218,64 @@ public void run() {
             return true;
         }
 
+
+        if ("overlaysWebView".equals(action)) {
```
Comment (luissilvaos) [inline src/android/StatusBar.java]: This condition is duplicated

**Review by luissilvaos — CHANGES_REQUESTED** [review]
Check my comments please

File: src/android/StatusBar.java (hunk 1 above)
Comment (pmaroco) [reply src/android/StatusBar.java]: Fixed!

**Review by vmfo22 — APPROVED** [review]

**Review by EiyuuZack — APPROVED** [review]

File: src/android/StatusBar.java [hunk 2]
```diff
@@ -112,38 +116,6 @@ public boolean execute(final String action, final CordovaArgs args, final Callba
...
-
-                    if (doOverlay) {
-                        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
-                            window.addFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);
-                            window.addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
-                        } else {
-                            LOG.e(TAG, "Translucent status bar not supported in your Android version");
-                        }
-
-                        ActivityAssistant.getInstance().applyGlobalLayoutListener();
```
Comment (luissilvaos) [inline src/android/StatusBar.java:134]: You miss the merge of this action and lost important code here.

**Review by luissilvaos — CHANGES_REQUESTED** [review]
Check my comments please.

**Review by luissilvaos — APPROVED** [review]

**Pull request merged**